packages/framework/theming-react-native/src/BaselinePlatformDefaults.ts

```diff
@@ -38,8 +40,22 @@ function _defaultTypography(): ITypography {
       heroSemibold: { face: 'primary', size: 'hero', weight: 'semiBold' },
       heroLargeStandard: { face: 'primary', size: 'heroLarge', weight: 'regular' },
       heroLargeSemibold: { face: 'primary', size: 'heroLarge', weight: 'semiBold' }
-    }
+    } as IVariants
   };
+
+  if (Platform.OS === 'macos' || Platform.OS === 'ios') {
+    const familiesDictApple = {
+      primary: 'System',
+      secondary: 'System',
+      cursive: 'System',
+      monospace: 'System',
+      sansSerif: 'System',
+      serif: 'System'
+    }
```

Comment:
Should this be "as IVariants"?

Reply:
I see why you say that because the win version is casted. First thoughts are to comply with the transpiler instead of explicitly casting everything which could end up with redundancies. I'm open to discuss it more online if you think we _should_ always cast in these scenarios.